Makefile

```diff
@@ -26,6 +27,9 @@ endif # TARGET
 ASSISTED_ORG := $(or ${ASSISTED_ORG},quay.io/ocpmetal)
 ASSISTED_TAG := $(or ${ASSISTED_TAG},latest)
 
+# IMAGE_FORMAT set by openshift ci to <registry>/<namespace>/stable:${component}
+component = assisted-service
```

Comment:
I don't see it is being used anywhere. Can be removed

Reply:
I hope my comment for `SERVICE` clarifies this point.